xdsl/dialects/builtin.py

```diff
@@ -397,21 +397,26 @@ def verify_value(self, value: int):
                 f"values in the range [{min_value}, {max_value})"
             )
 
-    def normalized_value(self, value: IntAttr) -> IntAttr | None:
+    def normalized_value(
+        self, value: IntAttr, *, truncate_bits: bool = False
```

Comment:
I think a name like `wrapping` would be clearer than `truncate_bits`.

Reply:
Hmm, what's it wrapping?

Reply:
Yes, and truncating has already a meaning in LLVM/arith, which is to go from let's say i32 to i16, which is not really what we are doing here

Reply:
But wrapping is a feature of the operation, not of a value normalization step. I'm happy for an addition to be wrapping, but it seems a bit out of context here.

Reply:
Here's the relevant API that I have most experience with https://developer.apple.com/documentation/swift/int/init(truncating:)

Reply:
Did we come to a conclusion?

Reply:
We can hash it out on the zulip if you want, I'm happy to be outvoted if you both think wrapping is more intuitive here